index.js

```diff
@@ -102,32 +102,33 @@ class DeltaChat extends EventEmitter {
       throw new Error('Missing .addr')
     }
 
-    if (typeof opts.mail_pw !== 'string') {
-      throw new Error('Missing .mail_pw')
+    if (typeof opts.mailPw !== 'string') {
+      throw new Error('Missing .mailPw')
     }
 
     this.once('_configured', ready)
 
-    if (typeof opts.e2ee_enabled === 'undefined') opts.e2ee_enabled = 1
+    if (typeof opts.e2eeEnabled === 'undefined') opts.e2eeEnabled = 1
 
     this.setConfig('addr', opts.addr)
 
-    this.setConfig('mail_server', opts.mail_server)
-    this.setConfig('mail_user', opts.mail_user)
-    this.setConfig('mail_port', String(opts.mail_port))
-
-    this.setConfig('mail_pw', opts.mail_pw)
-    this.setConfig('send_server', opts.send_server)
-    this.setConfig('send_user', opts.send_user)
-    this.setConfig('send_pw', opts.send_pw)
-    this.setConfig('send_port', String(opts.send_port))
-    this.setConfig('server_flags', String(opts.server_flags))
-    this.setConfig('displayname', opts.displayname)
-    this.setConfig('selfstatus', opts.selfstatus)
-    this.setConfig('selfavatar', opts.selfavatar)
-    this.setConfig('e2ee_enabled', String(opts.e2ee_enabled ? 1 : 0))
-    this.setConfig('mdns_enabled', String(opts.mdns_enabled ? 1 : 0))
-    this.setConfig('save_mime_headers', String(opts.save_mime_headers ? 1 : 0))
+    this.setConfig('mail_server', opts.mailServer)
+    this.setConfig('mail_user', opts.mailUser)
+    this.setConfig('mail_pw', opts.mailPw)
+    this.setConfig('mail_port', String(opts.mailPort))
+
+    this.setConfig('send_server', opts.sendServer)
+    this.setConfig('send_user', opts.sendUser)
+    this.setConfig('send_pw', opts.sendPw)
+    this.setConfig('send_port', String(opts.sendPort))
+
+    this.setConfig('server_flags', String(opts.serverFlags))
+    this.setConfig('displayname', opts.displayName)
+    this.setConfig('selfstatus', opts.selfStatus)
+    this.setConfig('selfavatar', opts.selfAvatar)
```

Comment:
Not testing this option yet. Will fix in a later commit.

Reply:
great!

Reply:
https://github.com/deltachat/deltachat-node/commit/bea037f6cbca782f8f42d29a0d4d93b007887c3d

However, `hq5.merlinux.eu` is currently down so tests fail atm.